hasher-matcher-actioner/scripts/create_test_bank.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+"""
+Create a bank and fill it with photos/videos for testing.
+"""
+
+import os
+from hmalib.common.models.models_base import DynamoDBItem
+from hmalib.common.models.bank import BanksTable, BankMember
+from threatexchange.content_type.video import VideoContent
+from mypy_boto3_dynamodb.service_resource import Table
+import boto3
+
+"""
+Replace this with the table to which you want to add data.
+If you don't already have a table, source the script set_tf_outputs_in_local_env.sh
+to create table. Name will be stored in environemnt variable BANKS_TABLE.
+"""
+
+dynamodb = boto3.resource("dynamodb")
+# add the name of your table here
+table_name = ''
```

Comment:
nit: totally fine with requiring script edits before running for simplicity but can you also make `num_members` a variable as well as `test_bank_name` instead of `"test bank"` and  `1000` respectively.

Also having an assert/conditional that exits and print a warning if you try to use this script without the edits would be appreciated